rocketpy/simulation/flight.py

```diff
@@ -2536,6 +2507,7 @@ def max_dynamic_pressure(self):
     # Total Pressure
     @funcify_method("Time (s)", "Total Pressure (Pa)", "spline", "zero")
     def total_pressure(self):
+        """Total pressure as a Function of time."""
```

Comment:
What is a total pressure? 

Do you see the equation? I don't completely get it

Reply:
I do not know where the expression comes from. I just added the missing docs